<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>621</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>39</width>
         <height>37</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(114, 159, 207);
border:solid rgb(114, 159, 207);
border-radius:6px;
}

QPushButton:hover{
background-color: rgb(179, 119, 0);
border:solid rgb(179, 119, 0);
border-radius:4px;
}
</string>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_5">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>37</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(114, 159, 207);
border:solid rgb(114, 159, 207);
border-radius:6px;
}

QPushButton:hover{
background-color: rgb(179, 119, 0);
border:solid rgb(179, 119, 0);
border-radius:4px;
}
</string>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_6">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>37</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(114, 159, 207);
border:solid rgb(114, 159, 207);
border-radius:6px;
}

QPushButton:hover{
background-color: rgb(179, 119, 0);
border:solid rgb(179, 119, 0);
border-radius:4px;
}
</string>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(114, 159, 207);
border:solid rgb(114, 159, 207);
border-radius:6px;
}

QPushButton:hover{
background-color: rgb(179, 119, 0);
border:solid rgb(179, 119, 0);
border-radius:4px;
}
</string>
       </property>
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>39</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(114, 159, 207);
border:solid rgb(114, 159, 207);
border-radius:6px;
}

QPushButton:hover{
background-color: rgb(179, 119, 0);
border:solid rgb(179, 119, 0);
border-radius:4px;
}
</string>
       </property>
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(114, 159, 207);
border:solid rgb(114, 159, 207);
border-radius:6px;
}

QPushButton:hover{
background-color: rgb(179, 119, 0);
border:solid rgb(179, 119, 0);
border-radius:4px;
}
</string>
    </property>
    <property name="text">
     <string>Добавить тс или собственника</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(114, 159, 207);
border:solid rgb(114, 159, 207);
border-radius:6px;
}

QPushButton:hover{
background-color: rgb(179, 119, 0);
border:solid rgb(179, 119, 0);
border-radius:4px;
}
</string>
    </property>
    <property name="text">
     <string>ВЫПИСАТЬ ШТРАФ</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
  background-color: #ffffff; 
  color: rgb(114, 159, 207);
  border-style: inherit;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-style:solid;
border-color:rgb(114, 159, 207);
border-radius:8px;
}

QLineEdit:hover{
   
border-color: rgb(179, 119, 0);
}

</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
  background-color: #ffffff; 
  color: rgb(114, 159, 207);
  border-style: inherit;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-style:solid;
border-color:rgb(114, 159, 207);
border-radius:8px;
}

QLineEdit:hover{
   
border-color: rgb(179, 119, 0);
}

</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
  background-color: #ffffff; 
  color: rgb(114, 159, 207);
  border-style: inherit;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-style:solid;
border-color:rgb(114, 159, 207);
border-radius:8px;
}

QLineEdit:hover{
   
border-color: rgb(179, 119, 0);
}

</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
  background-color: #ffffff; 
  color: rgb(114, 159, 207);
  border-style: inherit;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-style:solid;
border-color:rgb(114, 159, 207);
border-radius:8px;
}

QLineEdit:hover{
   
border-color: rgb(179, 119, 0);
}

</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>112</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>Главная таблица</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>Правила</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>ТС</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>Собственники</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="radioButton_5">
       <property name="text">
        <string>Нарушения</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="radioButton_6">
       <property name="text">
        <string>Постановление</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>37</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(114, 159, 207);
border:solid rgb(114, 159, 207);
border-radius:6px;
}

QPushButton:hover{
background-color: rgb(179, 119, 0);
border:solid rgb(179, 119, 0);
border-radius:4px;
}
</string>
       </property>
       <property name="text">
        <string>Поиск по номеру</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>37</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(114, 159, 207);
border:solid rgb(114, 159, 207);
border-radius:6px;
}

QPushButton:hover{
background-color: rgb(179, 119, 0);
border:solid rgb(179, 119, 0);
border-radius:4px;
}
</string>
       </property>
       <property name="text">
        <string>Поиск</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>37</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(114, 159, 207);
border:solid rgb(114, 159, 207);
border-radius:6px;
}

QPushButton:hover{
background-color: rgb(179, 119, 0);
border:solid rgb(179, 119, 0);
border-radius:4px;
}
</string>
       </property>
       <property name="text">
        <string>Поиск</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>АСК Штрафы ГИБДД</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Информационные сведения</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_2">
   <property name="text">
    <string>Сведения о программе</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
